estate/models/estate_property.py

```diff
@@ -0,0 +1,93 @@
+from odoo import api, fields, models
+from dateutil.relativedelta import relativedelta
+from datetime import datetime
+from odoo.exceptions import UserError
+
+
+class EstateProperty(models.Model):
+    _name = "estate_property"
+    _description = "Estate Property"
+    name = fields.Char("Title", required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(
+        copy=False, default=datetime.now() + relativedelta(months=3)
```

Comment:
for the default values it is get evaluated on server boot-up So without the `lambda` once you run the server it will take the date and after 1 week if you created new property it will take the 3 month from the server boot-up not 3 month from real today.
```suggestion
        copy=False, default=lambda self: datetime.now() + relativedelta(months=3)
```